lib/types.js

```diff
@@ -31,6 +31,13 @@ export type TransformProps = {
   },
 };
 
+export type ShadowProps = {
```

Comment:
Since I just merged the `typings` PR, can you also update the `index.d.ts`? You will need to merge the latest `master` branch.